sbin/build.sh

```diff
@@ -548,22 +548,77 @@ configureAlsaLocation() {
   fi
 }
 
+getFreeTypePerJdk8Tag() {
+  if [ -n "${BUILD_CONFIG[TAG]}" ]; then
+    local majorBuildVersion=$(echo "${BUILD_CONFIG[TAG]}" | sed "s/jdk8u//" | sed "s/-.*//")
+    if [ 0${majorBuildVersion} -lt 482 ] ; then
+      echo "old"
+    elif [ 0${majorBuildVersion} -gt 482 ] ; then
```

Comment:
Should this be gt and equal to?

Reply:
Can we just simplify the logic here to capture `8u<version>` and see if `<version>` is >=482 and then apply the freetype from source?  There seems to be excess logic here

Reply:
> Should this be gt and equal to?

Nope. The "eq" is handled in final else. More readable it is:
```
if  majorBuildVersion< 482  ; then  //always old
     return  "old"
 elif majorBuildVersion > 482] ; then    //always new
     return  "new"
else // that means ==
  decideByMinorBuildNumber()
fi
```

Reply:
@karianna  Thanx for eyball, highly appreciated!